stage0/layout.ld

```diff
@@ -334,6 +334,10 @@ SECTIONS {
         LONG(ADDR(.boot.ghcb))
         LONG(SIZEOF(.boot.ghcb))
         LONG(SEV_SECTION_UNMEASURED)
```

Comment:
nit: since the GHCB is no longer the only unmeasured page, perhaps update this items comment to be a bit clearer